src/pocket/PocketALBApplication.ts

```diff
@@ -737,103 +716,86 @@ export class PocketALBApplication extends Resource {
-            id: 'expression',
-            expression: 'errors/requests*100',
-            label: 'HTTP 5xx Error Rate',
-            returnData: true,
-          },
-        ],
-        comparisonOperator: 'GreaterThanOrEqualToThreshold',
-        evaluationPeriods: evaluationPeriods.http5xxError,
-        datapointsToAlarm:
-          alarmsConfig?.http5xxError?.datapointsToAlarm ??
-          evaluationPeriods.http5xxError,
-        threshold: alarmsConfig?.http5xxError?.threshold ?? 5,
-        insufficientDataActions: [],
-        alarmActions: alarmsConfig?.http5xxError?.actions ?? [],
-        okActions: alarmsConfig?.http5xxError?.actions ?? [],
-        tags: this.config.tags,
-        alarmDescription: 'Percentage of 5xx responses exceeds threshold',
-      },
-      {
-        alarmName: 'Alarm-HTTPResponseTime',
-        namespace: 'AWS/ApplicationELB',
-        metricName: 'TargetResponseTime',
-        dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
-        period: alarmsConfig?.httpLatency?.period ?? 300,
-        evaluationPeriods: evaluationPeriods.httpLatency,
-        datapointsToAlarm:
-          alarmsConfig?.httpLatency?.datapointsToAlarm ??
-          evaluationPeriods.httpLatency,
-        statistic: 'Average',
-        comparisonOperator: 'GreaterThanThreshold',
-        threshold: alarmsConfig?.httpLatency?.threshold ?? 300,
-        alarmDescription: 'Average HTTP response time exceeds threshold',
-        insufficientDataActions: [],
-        alarmActions: alarmsConfig?.httpLatency?.actions ?? [],
-        okActions: alarmsConfig?.httpLatency?.actions ?? [],
-        tags: this.config.tags,
-      },
-      {
-        alarmName: 'Alarm-HTTPRequestCount',
-        namespace: 'AWS/ApplicationELB',
-        metricName: 'RequestCount',
-        dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
-        period: alarmsConfig?.httpRequestCount?.period ?? 300,
-        evaluationPeriods: evaluationPeriods.httpRequestCount,
-        datapointsToAlarm:
-          alarmsConfig?.httpRequestCount?.datapointsToAlarm ??
-          evaluationPeriods.httpRequestCount,
-        statistic: 'Sum',
-        comparisonOperator: 'GreaterThanThreshold',
-        threshold: alarmsConfig?.httpRequestCount?.threshold ?? 500,
-        alarmDescription: 'Total HTTP request count exceeds threshold',
-        insufficientDataActions: [],
-        alarmActions: alarmsConfig?.httpRequestCount?.actions ?? [],
-        okActions: alarmsConfig?.httpRequestCount?.actions ?? [],
-        tags: this.config.tags,
-      },
-    ];
+    const defaultAlarms: CloudwatchMetricAlarmConfig[] = [];
+
+    if (alarmsConfig?.http5xxError) defaultAlarms.push(http5xxAlarm);
+
+    if (alarmsConfig?.httpLatency) defaultAlarms.push(latencyAlarm);
 
     if (alarmsConfig?.customAlarms) {
       defaultAlarms.push(...alarmsConfig.customAlarms);
     }
 
-    this.createAlarms(defaultAlarms);
+    if (defaultAlarms.length) this.createAlarms(defaultAlarms);
```

Comment:
do we need the `if` here? `createAlarms` does a `forEach`... doesn't hurt i guess.